src/SDK/Metrics/MetricMetadataInterface.php

```diff
@@ -22,7 +22,7 @@ public function description(): ?string;
     /**
      * Returns the underlying temporality of this metric.
      *
-     * @return string|Temporality internal temporality
+     * @return ?Temporality internal temporality
      */
-    public function temporality();
+    public function temporality(): ?Temporality;
```

Comment:
I wasn't sure about this, however there is a `lowmemory` test that fails if null is not allowed.

Reply:
Non-nullable is correct here - test should be fixed.

Reply:
Updated the offending test. `lowmemory` = "it depends on the instrument", [per spec](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.41.0/specification/metrics/sdk_exporters/otlp.md#additional-environment-variable-configuration) so I set a default and confirm that's used.